webapp/channels/src/components/post_view/floating_timestamp/floating_timestamp.test.tsx

```diff
@@ -13,10 +14,27 @@ describe('components/post_view/FloatingTimestamp', () => {
         toastPresent: true,
         isRhsPost: false,
     };
+    const initialState = {
+        entities: {
+            general: {
+                config: {},
+            },
+            preferences: {
+                myPreferences: {},
+            },
+        },
+    };
+
+    test('should match component state with given props', () => {
+        renderWithIntlAndStore(<FloatingTimestamp {...baseProps}/>, initialState);
+
+        const floatingTimeStamp = screen.getByTestId('floatingTimestamp');
+        const time = screen.getByText('January 01, 1970');
+
+        expect(floatingTimeStamp).toBeInTheDocument();
+        expect(floatingTimeStamp).toHaveClass('post-list__timestamp scrolling toastAdjustment');
```

Comment:
Is there a specific class we want to look for in this case? I'd just assume this component always has `post-list__timestamp`, so I feel like we could probably leave that one off, although since this test just used to be a snapshot with no extra details about what it's looking for, I don't think that matters too much

Reply:
`toastAdjustment` I am checking all the class